app/assets/stylesheets/markets.scss

```diff
@@ -0,0 +1,71 @@
+body {
+  background: url(/assets/Destruction.jpg);
+  background-size: cover;
```

Comment:
Properties should be ordered background, background-attachment, background-color, background-repeat, background-size